cmd/agent/app/secret.go

```diff
@@ -46,12 +46,12 @@ var secretInfoCommand = &cobra.Command{
 		}
 
 		if err := util.SetAuthToken(); err != nil {
-			fmt.Printf("%s", err)
+			fmt.Println(err)
```

Comment:
Note: In the general case, it is not equivalent (if err contain `%s`).

Reply:
If I understand correctly, `Println` does't parse format specifiers.